internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -430,25 +399,25 @@ func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(instance *ad
 		{Name: "SKIP_CERT_VERIFICATION", Value: "True"},
 
 		// database
-		newSecretEnvVar("DB_HOST", "db.host", databaseSecretName),
-		newSecretEnvVar("DB_NAME", "db.name", databaseSecretName),
-		newSecretEnvVar("DB_PASS", "db.password", databaseSecretName),
-		newSecretEnvVar("DB_PORT", "db.port", databaseSecretName),
-		newSecretEnvVar("DB_USER", "db.user", databaseSecretName),
+		newSecretEnvVar("DB_HOST", "db.host", databaseName),
+		newSecretEnvVar("DB_NAME", "db.name", databaseName),
```

Comment:
FWIW, I'm not a huge fan of using the default `postgres` database. I'd rather have a dedicated DB for the service.

Reply:
not dealing with this here.